pages/3_managetables.php

```diff
@@ -0,0 +1,77 @@
+<?php
+setTitle("Beheer tafels");
+
+
+if (isset($_POST['save_new'])) {
+    // Create a new table with the specified capacity
+    base_query("INSERT INTO `table` (Capacity) VALUES (:capacity)", [
+        ':capacity' => $_POST['capacity']
+    ]);
+}
+elseif (isset($_POST['save'])) {
+    // Update the specified table with the new capacity
+    base_query("UPDATE `table` SET Capacity = :capacity WHERE Id = :id", [
+        ':capacity' => $_POST['capacity'],
+        ':id' => $_POST['tableid']
+    ]);
+}
+elseif (isset($_POST['switch_status'])) {
+    // Deactivate the specified table
+    base_query("UPDATE `table` SET Activated = NOT Activated WHERE Id = :id", [
+        ':id' => $_POST['tableid']
+    ]);
+}
+
+$tables = base_query("SELECT * FROM `table` ORDER BY Activated DESC, Id DESC")->fetchAll();
+?>
+
+<h1>Beheer tafels</h1>
+
+<script>
+    // Function to enable or disable the button of a form if the the original value of the current element is different.
+    function updateSaveButton(elem) {
+        // Get the form of this button
+        var form = elem.form;
+
+        // Get the submit button of the form
+        var submitBtn = form.querySelector("[type=submit]")
+
+        // Disable the button if it's the same as it previously was.
+        submitBtn.disabled = elem.getAttribute("data-original") == elem.value;
+    }
+</script>
+
+<table class="table">
```

Comment:
Kun je ook toelichten, dmv comments, wat je in deze tabel, en in de komende tabellen, gaat doen?

Reply:
Welke komende tabellen?

Reply:
Fixed.